lib/Github/Api/PullRequest.php

```diff
@@ -58,6 +58,13 @@ public function files($username, $repository, $id)
         return $this->get('repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/pulls/'.rawurlencode($id).'/files');
     }
 
+    public function status($username, $repository, $id)
```

Comment:
yeah I didn't see it for the ones above so I wasn't sure if you were requiring it or not